### PR TITLE
[ENTESB-12114] Add backup and restore test

### PR DESCRIPTION
**Pull request by avano**
//skip-ci as our automation still runs against an old version (1.9.100-20191210)

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/OperatorValidationSteps.java [hunk 1]
```diff
@@ -282,4 +321,120 @@ public void createPv(String className) {
...
+            fail("Unable to create local backup file: ", e);
+        }
+        s3.downloadFile(S3BucketNameBuilder.getBucketName(SYNDESIS_BACKUP_BUCKET_PREFIX), fullFileName, backupTempFile);
+    }
+
+    @When("prepare backup folder")
+    public void prepareBackupFolder() {
+        try {
+            // There is some mess with access rights in docker when using createTempDirectory, so create temp directory manually
+            backupTempDir = Files.createDirectory(Paths.get("/tmp", "syndesis-backup-" + new Random(new Date().getTime()).nextInt()));
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/OperatorValidationSteps.java]: [Files.createTempDirectory](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory%28java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...%29)?

**Pull request merged**